Generals/Code/Libraries/Source/WWVegas/WW3D2/dx8wrapper.cpp

```diff
@@ -1470,6 +1539,11 @@ void DX8_Assert()
 void DX8Wrapper::Begin_Scene(void)
 {
 	DX8_THREAD_ASSERT();
+
+#if ENABLE_EMBEDDED_BROWSER
+	DX8WebBrowser::Update();
```

Comment:
Just Note: I wonder if this additional call is a bug. Because there is another call to `DX8WebBrowser::Update()` below.